### PR TITLE
Add to Astra Streaming guide

### PR DESCRIPTION
**Pull request by emile-00**
## Info
- **Description**: 
Add a more descriptive guide on how to connect to an Astra Streaming topic from RisingWave.

- **Preview**: 
https://pr-878.d2fbku9n2b6wde.amplifyapp.com/docs/upcoming/connector-astra-streaming/

- **Related code PR**: 
[ Provide the link to the code PR here. For example, https://github.com/risingwavelabs/risingwave/pull/4085. Delete this part if there's no code PR related. ]

- **Related doc issue**: 
Resolves [ Provide the link to the doc issue here. ]

- **Notes**: 
[ Any additional information? ]

## Before merging
  - [ ] (For version-specific PR) I have selected the corresponding software version in **Milestone** and linked the related doc issue to this PR in **Development**.
  - [ ] I have acquired the approval from the owner (and optionally the reviewers) of the code PR and at least one tech writer (`bernscode`, `CharlieSYH`, `emile-00`, & `hengm3467`). 
  - [ ] I have checked the doc site preview, and the updated parts look good. <details><summary>How?</summary>Scroll down and open this link: image
</details>

### CODE REVIEW

File: docs/guides/connector-astra-streaming.md [hunk 1]
```diff
@@ -5,11 +5,82 @@ description: Ingest data from DataStax Astra Streaming.
...
+{"v1":0,"v2":"name5"}
+{"v1":5,"v2":"name8"}
+```
+
+### Query the messages in RisingWave
+
+Now we can query the table in RisingWave to see that RisingWave has consumed the messages.
 
-To learn about how to set up an Astra Streaming account and create a topic, see the [Astra Streaming QuickStart](https://docs.datastax.com/en/streaming/astra-streaming/getting-started/index.html).
+```sql
```
Comment (CharlieSYH) [inline docs/guides/connector-astra-streaming.md:76]: The docs now support merging two code blocks (SQL code and response blocks).
Usage (replace `` with ```):

```
``sql
code
``
``
response
``
```

File: docs/guides/connector-astra-streaming.md [hunk 2]
```diff
@@ -5,11 +5,82 @@ description: Ingest data from DataStax Astra Streaming.
...
+
+### Produce messages in Astra Streaming
+
+We can now send messages from Astra Streaming to RisingWave.
+
+Navigate to the tenant RisingWave is connected to in Astra Streaming and click on the `Try me` tab. Ensure the `Namespace`, `Producer topic`, and the `Consumer topic` match the Astra Streaming topic that RisingWave is consuming data from. 
+
+Set the `Connection type` as `Read` and the `Read position` as `Earliest`. Click `Connect`.
+
+Try sending the following messages line by line in the `Test message` text box. Set the `Message type` as `JSON`. Note that the schema of the messages match the schema of the table we created in RisingWave.
```
Comment (CharlieSYH) [inline docs/guides/connector-astra-streaming.md]: Note that the schema of the messages **match**: matches?

**Review by CharlieSYH — APPROVED** [review]

**Review by Honeta — APPROVED** [review]

File: docs/guides/connector-astra-streaming.md [hunk 3]
```diff
@@ -5,11 +5,84 @@ description: Ingest data from DataStax Astra Streaming.
...
+   auth.token='replace me with your token'
+)
+ROW FORMAT JSON;
+```
+
+### Produce messages in Astra Streaming
+
+We can now send messages from Astra Streaming to RisingWave.
+
+Navigate to the tenant RisingWave is connected to in Astra Streaming and click on the `Try me` tab. Ensure the `Namespace`, `Producer topic`, and the `Consumer topic` match the Astra Streaming topic that RisingWave is consuming data from.
```
Comment (hengm3467) [inline docs/guides/connector-astra-streaming.md]: From the UI, it's `Try Me`

File: docs/guides/connector-astra-streaming.md [hunk 4]
```diff
@@ -5,11 +5,84 @@ description: Ingest data from DataStax Astra Streaming.
...
+
+## Consume data from Astra Streaming in RisingWave
+
+### Install and launch RisingWave
+
+See the [Get started](../get-started.md) guide for options on how you can run RisingWave.
+
+### Create a table in RisingWave
+
+To learn about the specific syntax used to consume data from a Pulsar topic, see [Ingest data from Pulsar](../create-source/create-source-pulsar.md). To learn about the specific syntax used to consume data from a Kafka topic, see [Ingest data from Kafka](../create-source/create-source-kafka.md).
```
Comment (hengm3467) [inline docs/guides/connector-astra-streaming.md:34]: Is Kafka related? I think Astra Streaming supports Pulsar only. Correct me if I'm wrong. @Honeta

**Review by hengm3467 — APPROVED** [review]
Otherwise looks good to me. Really solid work!

File: docs/guides/connector-astra-streaming.md (hunk 4 above)
Comment (Honeta) [reply docs/guides/connector-astra-streaming.md:34]: While Astra Streaming supports pulsar, it also provides the option to use Kafka to connect to the streaming tenant.

**Pull request merged**